src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -331,10 +324,19 @@ export default {
 			isTallEnough: false,
 			showReloadButton: false,
 			isDeleting: false,
+			// whether the message was seen, only used if this was marked as last read message
+			seen: false,
```

Comment:
actually now thinking, this only means that the marker was seen, not the full message's bounding box
so it could cause situations where we mark as seen when it's not

Reply:
it seems to work fine that way